code/Modules/Commands/ItemsCommands.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Sandbox;
+using ssl.Modules.Items;
+using ssl.Modules.Items.Carriables;
+using ssl.Player;
+
+namespace ssl.Modules.Commands
+{
+    public class ItemsCommands
+    {
+        /// <summary>
+        /// Clear the inventory
+        /// </summary>
+        [AdminCmd("clear")]
```

Comment:
```suggestion
        [AdminCmd("inv_clear")]
```

Reply:
I wanted to make easy commands for fast tasks. In mc it is clear

Reply:
yeah but won't it overwrite a "clear" command to clear the console ?

Reply:
I checked and there's already "clear" so we'll go for the prefix "inv"